### PR TITLE
[CI/Build][LoRA] Temporarily fix long context failure issue

### PR DESCRIPTION
**Pull request by jeejeelee**
#7049  added the parameter `disable_async_output_proc` with a default value of False, which led to `test_long_context` raising an `IndexError: list index out of range error`, see:  https://buildkite.com/vllm/ci-aws/builds/7645#01919198-199d-488c-9834-8c48ae675338/193-420.  Since then, this test has consistently shown this error, which has hidden errors introduced by other PRs. For example, when setting `disable_async_output_proc=True`, inference works without errors but generates incorrect results due to the cudagraph issue from #8645 (which has been fixed by #9549). 

This PR is a temporary solution to make this  test pass correctly, so that issues like #8645 can be exposed earlier in CI.

ping @DarkLight1337 , also cc  @megha95  @alexm-neuralmagic

### CODE REVIEW

File: tests/lora/test_long_context.py [hunk 1]
```diff
@@ -115,6 +115,7 @@ def lora_llm(long_context_infos):
                    long_lora_scaling_factors=tuple(scaling_factors),
                    max_num_batched_tokens=4096 * 8,
                    tensor_parallel_size=4,
+                   disable_async_output_proc=True,
```
Comment (DarkLight1337) [inline tests/lora/test_long_context.py:118]: Can you add a `FIXME` here as a reminder to fix the test when async output processor is enabled?

Comment (jeejeelee) [reply tests/lora/test_long_context.py]: Done, and please add `/ready` ,thanks~

**Review by DarkLight1337 — APPROVED** [review]
Thanks for fixing!

File: tests/lora/test_long_context.py [hunk 2]
```diff
@@ -92,10 +99,12 @@ def batched_generate(
     for input in inputs:
         prompt, sampling_param, lora_req = input
         # Add requests to the engine and run the engine
-        llm._validate_and_add_requests(prompt,
-                                       sampling_param,
-                                       lora_request=lora_req,
-                                       prompt_adapter_request=None)
```
Comment (DarkLight1337) [inline tests/lora/test_long_context.py:98]: Please avoid changing the whitespace unless you actually updated this code.

**Comment by jeejeelee** [comment]
Mark: `test_long_context.py` test passed,see: https://buildkite.com/vllm/ci-aws/builds/10183#0192b31a-7610-4295-b9e8-8c18fb417560

**Pull request merged**